pom.xml

```diff
@@ -800,6 +800,32 @@
         <version>${curator.version}</version>
         <scope>test</scope>
       </dependency>
+	  <dependency>
```

Comment:
This does seem like a lot of build complexity relative to its value.

BTW isn't this taking Hadoop or some of its dependencies out of compile scope? I am not sure why it would entail the changes to the deps files above